server/app/serializers/rpc/volume_serializer.rb

```diff
@@ -9,12 +9,11 @@ def initialize(volume_instance)
 
     def to_hash
```

Comment:
With the `RpcSerializer` these should just be `attribute ...` lines, see the `HostNodeSerializer` for an example. The `ServicePodSerializer` doesn't use that for some reason.